dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
 pandas
 selenium
-dash
+dash>=0.32.2
```

Comment:
Is this a new requirement for the non-dev package? If so should it be set in setup.py?

Reply:
No it's a requirement for building the components.

Reply:
Ok, thanks. I was seeing this failure https://circleci.com/gh/conda-forge/dash-html-components-feedstock/17?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link upstream for 0.13.3. SyntaxError for the call sig in Script.py. Thought this might be related. Should 0.13.3 build?

Reply:
Ok.